e2e_tests/conftest.py

```diff
@@ -7,6 +7,12 @@
 from config.settings.base import BASE_DIR
 
 
+@pytest.fixture(scope="session")
+def browser_context_args(browser_context_args):
+    # ignore complaints that our self-signed localhost cert isn't trusted by playwright
```

Comment:
Where is this self-signed cert? Do I have to create it?

Reply:
I accidentally already committed that when I did the nginx config rework, so it's been checked in for a while and doesn't show up in this PR.
It can be found here: [ssl_certificates_localhost](https://github.com/GermanZero-de/klimaschutzmonitor/tree/main/ssl_certificates_localhost)